adapters/Adapter.py

```diff
@@ -1,6 +1,6 @@
 # Author Abdulrahman S. Omar <xabush@singularitynet.io>
 
-class Adapter:
+class Adapter1:
```

Comment:
Why did we need to rename it to `Adapter1`? And again it is being used as `Adapter` in the `compound Adapter`.

Reply:
> Why did we need to rename it to `Adapter1`? And again it is being used as `Adapter` in the `compound Adapter`.

It was unintentional change. I have changed it back to Adapter.